lib/components/user/existing-account-display.js

```diff
@@ -1,43 +1,42 @@
-import React, { Component } from 'react'
+import React from 'react'
 
 import LinkButton from './link-button'
 import StackedPaneDisplay from './stacked-pane-display'
 
 /**
  * This component handles the existing account display.
  */
-class ExistingAccountDisplay extends Component {
-  render () {
-    const { onCancel, onComplete, panes } = this.props
-    const paneSequence = [
-      {
-        pane: () => <p><LinkButton to='/savedtrips'>Edit my trips</LinkButton></p>,
-        title: 'My trips'
-      },
-      {
-        pane: panes.terms,
-        props: { disableCheckTerms: true },
-        title: 'Terms'
-      },
-      {
-        pane: panes.notifications,
-        title: 'Notifications'
-      },
-      {
-        pane: panes.locations,
-        title: 'My locations'
-      }
-    ]
+const ExistingAccountDisplay = props => {
+  const { onCancel, panes } = props
+  const paneSequence = [
+    {
+      pane: () => <p><LinkButton to='/savedtrips'>Edit my trips</LinkButton></p>,
+      title: 'My trips'
+    },
+    {
+      pane: panes.terms,
+      props: { disableCheckTerms: true, ...props },
```

Comment:
This may not really be an issue, but would it be better to add `props` before the `disableCheckTerms`? It seems like we may not want this to ever be enabled for existing accounts, so it could be good to ensure that with the ordering of the spread operation/additional fields.